src/public/target/sections/Pathways/Section.js

```diff
@@ -14,7 +14,7 @@ const PathwaysDetail = ({ symbol, uniprotId, data }) => {
       id,
       name: label,
       parents: topLevelParents,
-      parentNames: topLevelParents.map(parent => parent.name),
+      parentNames: topLevelParents.map(parent => parent.name).join(', '),
```

Comment:
If this change causes any confusion: it's here because sorting should work just as well on a single string with commas as it did on an array of strings, whereas having a field with a single string means that we were able factor out the `getDownloadRows()` method that produced such a string.